scripts/run_docker_build.sh

```diff
@@ -20,16 +20,17 @@ CONDARC
 cat << EOF | docker run -i \
                         -v ${REPO_ROOT}:/staged-recipes \
                         -a stdin -a stdout -a stderr \
+                        -e HOST_USER_ID=`id -u $USER` \
                         $IMAGE_NAME \
-                        bash || exit $?
+                        bash -ex || exit $?
```

Comment:
I don't think we have any special tokens that will show up here, but we might want to double check before going down this road.

cc @pelson